ansible/roles/docs-rs-builder/templates/sudoers

```diff
@@ -0,0 +1,6 @@
+#
+# {{ ansible_managed }}
+#
+
+builder ALL= NOPASSWD: /bin/systemctl stop builder.service
+builder ALL= NOPASSWD: /bin/systemctl start builder.service
```

Comment:
Maybe allow restarting too? How are we going to manage the service in prod if we run into issues?